client/my-sites/themes/theme-preview.jsx

```diff
@@ -1,39 +1,26 @@
-import { isEnabled } from '@automattic/calypso-config';
 import { Button } from '@automattic/components';
-import { getDesignPreviewUrl } from '@automattic/design-picker';
-import { createHigherOrderComponent } from '@wordpress/compose';
 import { localize } from 'i18n-calypso';
-import page from 'page';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
 import { connect } from 'react-redux';
 import QueryCanonicalTheme from 'calypso/components/data/query-canonical-theme';
-import PremiumGlobalStylesUpgradeModal from 'calypso/components/premium-global-styles-upgrade-modal';
```

Comment:
Should we remove this component from the code base as well? 🤔

Reply:
Do you mean `PremiumGlobalStylesUpgradeModal`? If so, we can remove it and add it back when we need it in the future 🙂

Reply:
Yep! Both `PremiumGlobalStylesUpgradeModal` and `ThemePreviewModal` were created for this new preview modal. Now that we are reverting it, we can remove it from the code as well.